### DISCUSSION TITLE
Disconnect from STA SSID

### DISCUSSION THREAD

**Discussion by Grutsch** [Hieromon/AutoConnect#463] [Q&A]
Is there a way to disconnect from the saved STA SSID? I tried auto reconnect but still will connect to SSID. What I am trying to achieve is that I can switch between AP and STA Mode even when the access point is still available.

**Comment by Hieromon** [comment]
@Grutsch 
> I can switch between AP and STA Mode even when the access point is still available.

The above behavior you are looking for is ambiguous.
But, you can use [AutoConnectConfig::autoRise and immediateStart](http://localhost:8000/AutoConnect/adcpcontrol.html#captive-portal-start-control) the ESP module always opens the captive portal, even if the ESP finds an AP to which it can connect automatically.
